index.js

```diff
@@ -75,18 +75,16 @@ class ClientServiceBase {
   }
 
   /**
-   * get max age
-   * INT32_MAX is equivalent to
-   * java's Integer.MAX_VALUE which is 2147483647
+   * Get ledger asset maximum age which is equivalent to
```

Comment:
I think `equivalent` sounds a bit weird since they are the same value.
Maybe just saying `the same` or `consistent`.

Reply:
In this case, equivalent means "the same" or "equal to", so I think it's okay